README.md

```diff
@@ -11,6 +11,10 @@ The current architecture uses certain retrieval techniques on OpenROAD documenta
 
 ## Setup
 
+### Building Manpages
+
+Build manpages as per the instructions [here](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/docs). Place the markdown files in `backend/data/markdown/manpages` before proceeding.
```

Comment:
Maybe automate this step if possible? Or add a CLI option while building your docker image